README.md

```diff
@@ -7,40 +7,33 @@
 MNE-BIDS
 ========
 
-This is a repository for creating BIDS-compatible datasets with MNE.
+This is a repository for creating [BIDS](http://bids.neuroimaging.io/)-compatible datasets with [MNE](https://mne-tools.github.io/stable/index.html).
 
 Installation
 ------------
 
-We recommend the
-[Anaconda Python distribution](https://www.continuum.io/downloads).
-Next to `numpy`, `scipy`, and `matplotlib` that are included in the standard
-anaconda distribution, you will need to install the following dependencies
-to be able to use `mne_bids`:
+We recommend the [Anaconda](https://www.anaconda.com/download/) Python distribution.
+Besides `numpy`, `scipy`, `matplotlib`, and `pandas` (which are included in the standard
+Anaconda installation), you will need to install MNE using the `pip` tool:
 
-    $ pip install pandas
-    $ pip install -U https://api.github.com/repos/mne-tools/mne-python/zipball/master
+    $ pip install -U mne
```

Comment:
indeed, MNE master is required at least until 0.17 is released.

Reply:
OK, I'll add a note and revert this line then.